README.md

```diff
@@ -1,3 +1,5 @@
+# Monero Core GUI
```

Comment:
How about just **Monero GUI**?

I've gotten used to it, but it is kind of confusing that the GUI repo is called `monero-core` and the monero core project is called `monero`. Might as well be clear in the README about what's what :)